setup.py

```diff
@@ -75,13 +75,15 @@
         'libsass==0.14.2',  # 0.14.3 upwards changes imports for .css
         'lxml>=3.4.1',
         'mammoth>=1.4.4',
+        'pandas>=1.1.4',
```

Comment:
If this is only required for tests, it should go under the `test` section of `extras_require` below. Pandas is chunky to install.